templates/cloudflare-workers/app/entry.server.tsx

```diff
@@ -1,4 +1,4 @@
-import type { EntryContext } from "@remix-run/cloudflare";
+import { type EntryContext } from "@remix-run/cloudflare";
```

Comment:
Let's keep the type at the beginning of the import to keep it consistent with the rest of the codebase

```suggestion
import type { EntryContext } from "@remix-run/cloudflare";
```